cloudslang-runtime/src/main/java/io/cloudslang/lang/runtime/bindings/Binding.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * ****************************************************************************
+ * (c) Copyright 2014 Hewlett-Packard Development Company, L.P.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License v2.0 which accompany this distribution.
+ * <p/>
+ * The Apache License is available at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p/>
+ * *****************************************************************************
+ */
+package io.cloudslang.lang.runtime.bindings;
+
+import java.io.Serializable;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * @author Bonczidai Levente
+ * @since 11/6/2015
+ */
+public abstract class Binding {
+
+    // match ${ expression } pattern
+    private final static String EXPRESSION_REGEX = "^\\s*\\$\\{(.+)\\}\\s*$";
```

Comment:
It seems to me that the new constants: `ScoreLangConstants.EXPRESSION_START_DELIMITER` and the other one should be used here as part of the pattern, so that changing it in the constants will actually change it here